config.yml

```
@@ -39,7 +39,7 @@ badge_text: PROGRAM EDITION
 
 # Resource validator options: must be one of "internal" or "external". external_resource_validator_url is only used if resource_validator is set to external.
 resource_validator: external
-external_resource_validator_url: http://validator_service:4567
+external_resource_validator_url: http://localhost:8888
```

Looks like this was committed by mistake.

```suggestion
external_resource_validator_url: http://validator_service:4567
```

---

Ahhh. How could I get this file checked in again!